index.md

```diff
@@ -63,6 +63,7 @@ We will implement the [same wireframes](https://www.drupal.org/node/2818741#comm
 | React     | In progress  | Not yet | [Repo](https://github.com/contentacms/contenta_react) |
 | Vue.js + Nuxt.js | In progress  | [Website](https://contentavuenuxt.github.io/) | [Repo](https://github.com/contentacms/contenta_vue_nuxt) |
 | Chatbot   | Running demo  | [Blog post](https://medium.com/@dawehner/im-hungry-but-i-can-t-decide-a-small-chatbot-based-upon-contenta-cms-9f8c0bb1d48f) | [Repo](https://github.com/contentacms/contenta-bot) |
+| Gatsbyjs   | In progress | Not yet | [Repo](https://github.com/tarekdj/gatsby-contenta) |
```

Comment:
That's great news!!

Would you consider developing it in the contenta GitHub organization? Then the URL would be `https://github.com/contentacms/gatsby_contenta`. I think that will help potential collaborators find the project easier.

Reply:
@e0ipso of course that would ne great! I'll change the ownership of the repos ans update PR

Reply:
@e0ipso I need permission to create repos on contentacms.